config/routes.rb

```diff
@@ -692,7 +692,9 @@ def call(env)
       resource :settings, only: [:show, :update]
 
       namespace :registry, defaults: { format: :json } do
-        resources :policies, only: :create
+        constraints(id: /((?!\.json\Z|\.xml\Z)[^\/])+/) do
```

Comment:
We said we wouldn't add new paths supporting `xml`. And actually we respond only to json in this one:
https://github.com/3scale/porta/blob/35b4fb3fb6e2cbbee19dad25a44f724d5c2b107a/app/controllers/admin/api/registry/policies_controller.rb#L7

Reply:
This code smells anyway. It's really easy for one to add `respond_to :xml` and completely neglect this constraint. I think we should keep it this way for now until we have time to a better fix.

Reply:
>  It's really easy for one to add respond_to :xml and completely neglect this constraint

It is easy but a bad idea 😂 that would mean intentionally supporting xml and we don't want that 😄 


> I think we should keep it this way for now until we have time to a better fix.

That's fine 😄 It is not harmful anyway 👍